jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -177,6 +177,7 @@ YUI.add('user-profile-model-list', function() {
           return null;
         }
       }
+      const lastConnection = model.lastConnection || '--';
```

Comment:
Yeah, I kinda debated this. In the end, I decided that different clients (i.e. the components calling DateDisplay) might want to handle null values differently. Here in ModelList, we display '--' but elsewhere we might want to spit out an error. A third place we might want to just return an empty string. Consequently I decided to leave that functionality out of `DateDisplay`.